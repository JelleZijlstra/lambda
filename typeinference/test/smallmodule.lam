let foo = module
	interface { foo : int }

	let foo = 42;;

	print 3
end in
foo
