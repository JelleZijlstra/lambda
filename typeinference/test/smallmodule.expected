%%stdout%%
3
Result: module
	let __result__ = module
	let foo = 42
	let __result__ = ()
end
end
%%stderr%%
